ethicml/algorithms/preprocess/calders.py

```diff
@@ -75,7 +75,7 @@ def _calders_algorithm(
     data: Dict[Tuple[int, int], DataTuple] = {}
     for s, y in groups:
         s_y_mask = (dataset.s == s) & (dataset.y == y)
-        data[(s, y)] = DataTuple(
+        data[(s, y)] = DataTuple.from_x_s_and_y(
```

Comment:
```suggestion
        data[(s, y)] = DataTuple.from_x_s_and_y(
```
Maybe drop the 'and' for the sake of brevity?

Reply:
done.